src/Cohesity.Powershell/Cmdlets/Recovery/RestoreCohesityMSSQLObject.cs

```diff
@@ -264,6 +264,20 @@ protected override void ProcessRecord()
                 }
             };
 
+            if(false == JobRunId.HasValue || false == StartTime.HasValue)
```

Comment:
this should be an '&&' operation instead of '||' right?, if both are not given then select the latest run, if one of them is given for example jobRunId is given we restore from that run, if startTime is given get the run with that startTime and use it for restore.